File: practice-code-purple/c-programming/if-else.c
#include<stdio.h>

int main()
{
  int number;
  printf("Please Enter a number : \n");
  
  scanf("%d",&number);

  if(number < 100)
   printf("Number is less than 100\n");
  else if(number == 100)
   printf("Number is 100\n");
  else 
   printf("Number is greater than 100\n");
  return 0;    
}

File: pro-language-droid/csharp-prog/if-else.cs
using System;
namespace ifelse{
    class Program{
        static void Main(string[] args){
        int a = 50;
        
        if (a<100){
         Console.WriteLine("a is less than 100");
        }
        else{
          Console.WriteLine("a is not less than 100");
        }
         
        Console.WriteLine("value of a is : {0}",a);
        }
    }
}